plugins/ooni/main.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "OONI",
+    "name": "ooni",
```

Comment:
It should *not* be reverted. It will however be fixed when a fix for #15 is implemented. Basically NM doesn't handle multiple installs well, as well as folder names with same letters but different case ("ooni" vs "OONI")